### ISSUE TITLE
Issue on sameAs link in task-1 training set

### ISSUE THREAD

**Issue by jplu** [anuzzolese/oke-challenge#10]
Hi,

I saw a new bug in the dataset:

```
<http://www.ontologydesignpatterns.org/data/oke-challenge/task-1/St._Mark's_School>
        a           dul:Organization , owl:Individual ;
        rdfs:label  "St. Mark's School"@en ;
        owl:sameAs  dbpedia: .
```

Should be:

```
<http://www.ontologydesignpatterns.org/data/oke-challenge/task-1/St._Mark's_School>
        a           dul:Organization , owl:Individual ;
        rdfs:label  "St. Mark's School"@en .
```

Unless there is a DBpedia resource, but I didnt find it.

Cheers.

**Comment by giusepperizzo** [comment]
curl http://dbpedia.org/resource/St._Mark\'s_School -> 303
chromium http://dbpedia.org/resource/St._Mark\'s_School -> "No further information is available. (The requested entity is unknown)"

But if you describe it via SPARQL:
describe  <http://dbpedia.org/resource/St._Mark's_School>

it exists. So far, it's not the 1st case I've been experiencing. 

@anuzzolese @AnLiGentile will you still consider this as a valid disambiguation resource?

**Comment by MichaelRoeder** [comment]
Hi all,

just my two cents: there seem to be more than one single St. Mark's Schools. The sentence starts with

```
At St. Mark's School in Southborough, Massachusetts, ...
```

This school should be reachable using the URI `http://dbpedia.org/resource/St._Mark's_School_(Massachusetts)`, since it is referenced by the city `http://dbpedia.org/resource/Southborough,_Massachusetts` using this URI. However, this resource does not seem to exist.

Cheers
Michael

**Comment by rtroncy** [comment]
Actuallly, http://dbpedia.org/resource/St._Mark's_School_(Massachusetts) does exist, but don't trust your browser! There is a problem in how Openlink dereferences URIs that contain characters such as '. Do instead a:

```
DESCRIBE <http://dbpedia.org/resource/St._Mark's_School_(Massachusetts)>
```

and you will get the description of this resource

**Comment by MichaelRoeder** [comment]
Yes, the SPARQL query is working. Thanks for this hint!
Thus, from my point of view, it should be

```
<http://www.ontologydesignpatterns.org/data/oke-challenge/task-1/St._Mark's_School>
        a           dul:Organization , owl:Individual ;
        rdfs:label  "St. Mark's School"@en ;
        owl:sameAs  dbpedia:St._Mark's_School_(Massachusetts) .
```

**Comment by anuzzolese** [comment]
Hi all,

thanks for your precious hints.
dbpedia:St._Mark’s_School is actually a redirect, hence dbpedia:St._Mark's_School_(Massachusetts) is correct.
This is going to be fixed in the data.

## 

Andrea

**Comment by rtroncy** [comment]
@anuzzolese Are you saying we should expect a new training dataset anytime soon? Will you provide as well a diff log of what has been fixed?

**Comment by anuzzolese** [comment]
@rtroncy the training set has been already updated. In such an update

<http://www.ontologydesignpatterns.org/data/oke-challenge/task-1/St._Mark's_School>
        a           dul:Organization , owl:Individual ;
        rdfs:label  "St. Mark's School"@en ;
        owl:sameAs  dbpedia: .

has been changed to 

<http://www.ontologydesignpatterns.org/data/oke-challenge/task-1/St._Mark's_School>
        a           dul:Organization , owl:Individual ;
        rdfs:label  "St. Mark's School"@en ;
        owl:sameAs  dbpedia:St._Mark's_School_(Massachusetts) .
